src/main/java/com/minecolonies/coremod/network/messages/server/BuildToolPlaceMessage.java

```diff
@@ -104,6 +105,8 @@ public BuildToolPlaceMessage(
         this.state = state;
     }
 
+    public BlockPos getPos() { return pos; }
```

Comment:
potentially this properly formatted too

Reply:
To be fair. I think it is not a good idea to give the window the message. Properly better to just give the window all the data it needs in the constructor instead of giving it the message and adding a getter here.

Reply:
Fair enough, I was tossing that one up anyway. Personally I prefer passing the object to save the long argument list, especially seeing that we still need to construct the message anyway. This way the message is being instantiated in a single place. I could also just pass in the builder BlockPos as a single extra argument to the Window?

Reply:
that also works